timeseries/src/autogluon/timeseries/models/gluonts/torch/models.py

```diff
@@ -0,0 +1,167 @@
+        # which breaks training
+        self.callbacks = []
+        return super().save(path, **kwargs)
+
+    @classmethod
+    def load(cls, path: str, reset_paths: bool = True, verbose: bool = True) -> "AbstractGluonTSModel":
+        with torch_warning_filter():
+            model = load_pkl.load(path=path + cls.model_file_name, verbose=verbose)
+            if reset_paths:
+                model.set_contexts(path)
+            model.gts_predictor = GluonTSPyTorchPredictor.deserialize(Path(path) / cls.gluonts_model_path)
+        return model
+
+
+class DeepARModel(AbstractGluonTSPyTorchModel):
+    gluonts_estimator_class: Type[GluonTSPyTorchLightningEstimator] = DeepAREstimator
+
+
+class SimpleFeedForwardModel(AbstractGluonTSPyTorchModel):
+    gluonts_estimator_class: Type[GluonTSPyTorchLightningEstimator] = SimpleFeedForwardEstimator
+    float_dtype: Type = np.float32
+    int_dtype: Type = np.int32
+
+    def _get_estimator_init_args(self) -> Dict[str, Any]:
+        init_kwargs = super()._get_estimator_init_args()
+
+        # FIXME: PyTorch StudentT does not implement quantile functions
+        if "distr_output" in init_kwargs:
+            warnings.warn(
+                f"distr_output {init_kwargs['distr_output']} specified for SimpleFeedForward, however training"
+                "will default to the Gaussian distribution."
+            )
+        init_kwargs["distr_output"] = NormalOutput()
+        return init_kwargs
+
+    def _gluonts_forecasts_to_data_frame(
+        self, forecasts: List[Forecast], quantile_levels: List[float]
+    ) -> TimeSeriesDataFrame:
+        assert isinstance(forecasts[0], DistributionForecast)
+
+        result_dfs = []
+        for i, forecast in enumerate(forecasts):
+            item_forecast_dict = dict(mean=forecast.mean)
+            if isinstance(forecast.distribution, AffineTransformed):
+                # FIXME: this is a hack to get around GluonTS not implementing quantiles for
+                # torch AffineTransformed. We hence force PyTorch SFF to always use Gaussian error.
+                # However, this leads to a ~2x regression in error compared to MXNet SFF.
+                fdist = forecast.distribution
+                quantiles_tensor = torch.tensor(quantile_levels, device=fdist.scale.device).unsqueeze(1)
+                q_transformed = (
+                    (fdist.scale * fdist.base_dist.icdf(quantiles_tensor) + fdist.loc).cpu().numpy().tolist()
+                )
+                for ix, quantile in enumerate(quantile_levels):
+                    item_forecast_dict[str(quantile)] = q_transformed[ix]
+            else:
+                for quantile in quantile_levels:
+                    item_forecast_dict[str(quantile)] = forecast.quantile(str(quantile))
+
+            df = pd.DataFrame(item_forecast_dict)
+            df[ITEMID] = forecast.item_id
+            df[TIMESTAMP] = pd.date_range(
```

Comment:
Can you add a TODO to replace this with https://github.com/awslabs/autogluon/pull/2258/files#diff-0dcbec1000b6cda6a58a3e1c4cb6b5e8639942b0442f716e7f74d965215af35dR4?